on-prem-docs/separating-backend-services.md

```diff
@@ -0,0 +1,55 @@
+# Separating Backend Services (minio/postgresql)
+
+The on-prem-builder install.sh script now supports separation of the backend components i.e datastore and minio server onto different nodes.
+You can now have a setup where the postgresql service runs on one node and minio runs on another. Both can be designed to communicate to one another and to a third or more nodes running the front-end services. 
+
+## Pre-requisites
+The bldr.env file for your single on-prem builder node contains most of the information required setup minio and postgresql on it and will be used during the installation process.
```

Comment:
he bldr.env file contains all of the information required to setup minio and postgresql and will be used during the installation process.